src/views/Profile/Profile.js

```diff
@@ -0,0 +1,169 @@
+/* global BigInt */
+import React from 'react';
+import {
+  cssToReactStyleObject,
+  toniqFontStyles,
+  Icp16Icon,
+  LoaderAnimated24Icon,
+} from '@toniq-labs/design-system';
+import {useInterval} from '../../utilities/use-interval';
+import {profileStyles} from './ProfileStyles';
+import {ProfileHeader} from './ProfileHeader';
+import {ProfileBody} from './ProfileBody';
+import {ToniqIcon} from '@toniq-labs/design-system/dist/esm/elements/react-components';
+import {startLoadingProfileNftsAndCollections} from './ProfileLoadNfts';
+import {ProfileTabs, ProfileViewType} from './ProfileTabs';
+import {resolvedOrUndefined} from '../../utilities/async';
+import {useParams, useNavigate, useSearchParams, createSearchParams} from 'react-router-dom';
+import {spreadableSearchParams} from '../../utilities/search-params';
+
+const profileSearchParamName = 'profile-search';
+
+export function Profile(props) {
+  const classes = profileStyles();
+  const navigate = useNavigate();
+  const {tab: currentTab, address} = useParams();
```

Comment:
```suggestion
  const {tab: currentTab = ProfileTabs.MyNfts, address} = useParams();
```

If we go to `/profile` without params, there's no default tab selected and the params in the url is undefined hence it will take forever to load
![Screen Shot 2022-09-27 at 10 56 19 AM](https://user-images.githubusercontent.com/99699764/192515045-c579a848-b030-45a9-a287-e34bf99874f7.png)

Reply:
nice catch!